app/src/main/java/com/example/toDoListKotlin/ui/screens/listScreen/ListViewModel.kt

```diff
@@ -1,27 +1,31 @@
 package com.example.toDoListKotlin.ui.screens.listScreen
 
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.example.toDoListKotlin.dto.ToDoList
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
 
 class ListViewModel: ViewModel() {
-    private val _toDoLists: MutableLiveData<ArrayList<ToDoList>> = MutableLiveData(ArrayList())
-    val toDoLists: LiveData<ArrayList<ToDoList>> = _toDoLists
+    private val toDoLists: ArrayList<ToDoList> = ArrayList()
 
-    init {
-        for (i in 1..10) insertNewList()
+    val listFlow: Flow<ArrayList<ToDoList>> = flow {
+        while(true) {
+            repeat(10) {
+                toDoLists.add(generateNewList())
+            }
+
+            emit(toDoLists)
+            delay(1000)
+        }
     }
```

Comment:
Я тебе предлагаю сделать это
```suggestion
    val listFlow: Flow<ArrayList<ToDoList>> = flow {
            repeat(10) {
                toDoLists.add(generateNewList())
                
                emit(toDoLists)
                delay(1000)
            }
        }
    }
```

Reply:
Так каждую секунду будет добавляться и отображаться новый элемент